scispacy/custom_sentence_segmenter.py

```diff
@@ -5,24 +5,8 @@
 
 from spacy.tokens import Doc
 
-from scispacy.consts import ABBREVIATIONS # pylint: disable-msg=E0611,E0401
-
-def merge_segments(segments: List[str]) -> List[str]:
-    adjusted_segments: List[str] = []
-    temp_segment = ""
-    for segment in segments:
-        if temp_segment != "":
-            temp_segment += " "
-        temp_segment += segment
-        # add a space to each abbreviation so we only match it at the end of a sentence
-        if not segment.endswith(tuple([" " + abbreviation for abbreviation in ABBREVIATIONS])):
-            adjusted_segments.append(temp_segment)
-            temp_segment = ""
-
-    # on the off chance an abbreviation ends a sentence and is the end of the document, we just add that segment
-    if temp_segment != "":
-        adjusted_segments.append(temp_segment)
-    return adjusted_segments
+from scispacy.consts import ABBREVIATIONS  # pylint: disable-msg=E0611,E0401
+
 
 def combined_rule_sentence_segmenter(doc: Doc) -> Doc:
     """Adds sentence boundaries to a Doc. Intended to be used as a pipe in a spaCy pipeline.
```

Comment:
Could you add to the docstring here to just mention that we leveraging `pysbd` with a link to your repo?